rmw_fastrtps_dynamic_cpp/src/subscription.cpp

```diff
@@ -103,14 +103,23 @@ create_subscription(
 
   /////
   // Get RMW Type Support
+  auto type_hash = rosidl_get_zero_initialized_type_hash();
```

Comment:
```suggestion
  rosidl_type_hash_t type_hash = rosidl_get_zero_initialized_type_hash();
```

(also `#include "rosidl_runtime_c/type_hash.h"` at the top of this file)